app/models/member.rb

```diff
@@ -45,6 +62,29 @@ def self.search(search)
     end
   end
 
+  def self.by_gender
+    select("gender as label, count(*) as value").group("gender")
+  end
+
+  def age
+    now = Time.now.utc.to_date
```

Comment:
Dos cosas:

- En proyectos rails cuando usas la clase `Time` en general es mejor usar `Time.zone.now` o  `Time.current` en vez de `Time.now`, porque este último provoca muchos problemas cuando empiezas a trabajar con zonas horarias, ya que toma la zona horaria de la máquina en la que está corriendo y no UTC que es el estandar para guardar datos de tipo datetime.
- Al final el time lo estás transformando a un date. Eso mismo lo logras con `Date.current`
```
:009 > Time.now.utc.to_date
 => Sat, 04 Jun 2016 
 :010 > Date.current
 => Sat, 04 Jun 2016 
```
Échale una mirada a [este link](https://robots.thoughtbot.com/its-about-time-zones#a-summary-of-do39s-and-don39ts-with-time-zones), ahí sale explicado cuales son preferibles usar y cuales no.